source/api/commands/get.md

```diff
@@ -47,11 +47,12 @@ When using aliases with DOM elements, Cypress will query the DOM again if the pr
 
 Pass in an options object to change the default behavior of `cy.get()`.
 
-| Option    | Default                                                  | Description                        |
-| --------- | -------------------------------------------------------- | ---------------------------------- |
-| `log`     | `true`                                                   | {% usage_options log %}            |
-| `timeout` | {% url `defaultCommandTimeout` configuration#Timeouts %} | {% usage_options timeout cy.get %} |
-| `withinSubject` | null                                               | {% usage_options withinSubject %} |
+Option | Default | Description
+--- | --- | ---
+`log` | `true` | {% usage_options log %}
+`timeout` | {% url `defaultCommandTimeout` configuration#Timeouts %} | {% usage_options timeout cy.get %}
+`withinSubject` | null | {% usage_options withinSubject %}
+`includeShadowDom` | `false` | {% usage_options includeShadowDom %}
```

Comment:
technically, isn't the default Cypress.config.includeShadowDom(), for all of the commands?

```suggestion
`includeShadowDom` | value of `includeShadowDom` config variable | {% usage_options includeShadowDom %}
```